src/main/java/org/broadinstitute/hellbender/cmdline/programgroups/RNAProgramGroup.java

```diff
@@ -0,0 +1,14 @@
+package org.broadinstitute.hellbender.cmdline.programgroups;
+
+import org.broadinstitute.barclay.argparser.CommandLineProgramGroup;
+import org.broadinstitute.hellbender.utils.help.HelpConstants;
+
+/**
+ * Created by gauthier on 11/28/17.
+ */
+public final class RNAProgramGroup implements CommandLineProgramGroup {
```

Comment:
This is not a category. The tools under it are dispersed to other categories now. Thanks for the initiative in this though. Appreciate it.